File: FOP/java/factorial.java
import java.util.Scanner;

class Main{
  public static void main(String[] args){
    Scanner scan = new Scanner(System.in);
    int i=1, fact=1;
    
    System.out.print("Enter you number: ");
    int number = scan.nextInt();

    while (i<=number){
      fact = fact * i;
      i=i+1;
    }

    System.out.println("The factorial is: "+fact);
    
  }
}

File: FOP/c++/factorial.cpp
#include <iostream>
using namespace std;

int main(){
  int fact=1, i=1, user_input;
  cout << "Please enter the number: ";
  cin >> user_input;
  while (i<=user_input){
    fact = fact * i;
    i = i+1;
  }
  cout << "Factorial is: " << fact;
}